src/ngx_http_lua_bodyfilterby.c

```diff
@@ -86,12 +86,16 @@ ngx_int_t
 ngx_http_lua_body_filter_by_chunk(lua_State *L, ngx_http_request_t *r,
     ngx_chain_t *in)
 {
-    ngx_int_t        rc;
-    u_char          *err_msg;
-    size_t           len;
+    ngx_int_t                    rc;
+    ngx_chain_t                 *cl;
+    u_char                      *err_msg;
+    size_t                       len;
 #if (NGX_PCRE)
-    ngx_pool_t      *old_pool;
+    ngx_pool_t                  *old_pool;
 #endif
+    ngx_http_lua_ctx_t          *ctx;
+
+    ctx = ngx_http_get_module_ctx(r, ngx_http_lua_module);
```

Comment:
@agentzh fix.
I think it will add blank lines to mix into the following code, so I forced all of its variable declaration